pkg/resources/sku/sku.go

```diff
@@ -178,27 +190,32 @@ func (p AProductConfig) mutate(podTemplateSpec *v13.PodTemplateSpec, name string
 		if &container.Resources == nil {
 			podTemplateSpec.Spec.Containers[i].Resources = v13.ResourceRequirements{}
 		}
-		if container.Resources.Limits == nil {
-			podTemplateSpec.Spec.Containers[i].Resources.Limits = make(map[corev1.ResourceName]resource.Quantity)
-		}
+		checkResourceBlock(&container.Resources)
 		p.mutateResources(podTemplateSpec.Spec.Containers[i].Resources.Limits, resources.Limits)
-		if container.Resources.Requests == nil {
-			podTemplateSpec.Spec.Containers[i].Resources.Requests = make(map[corev1.ResourceName]resource.Quantity)
-		}
 		p.mutateResources(podTemplateSpec.Spec.Containers[i].Resources.Requests, resources.Requests)
 	}
 }
 
 func (p AProductConfig) mutateResources(pod, cfg v13.ResourceList) {
```

Comment:
The go syntax allows more than one function paramater to be defined of which they'll take the next defined type rather than redifining the type more than once.